src/pages/Dashboard.vue

```diff
@@ -221,6 +221,16 @@ function handleKeyboardShortcuts(e: KeyboardEvent) {
     e.preventDefault()
     return true
   }
+
+  // 'Search' => Search view
+  if (e.key === '/') {
```

Comment:
lgtm, was just wondering if we should include the ctrl/command key to that keybind

Reply:
Have to use shift+7 already on my current keyboard to get "/" so would be a bit wonky with ctrl+shift+7. Just / is the same shortcut in Gmail. Github supposedly also have "/" but that one doesn't work on my current layout only the alternate keyboard shortcut "s" works.